addons/stock_account/report/report_stock_forecasted.py

```diff
@@ -14,16 +14,17 @@ def _compute_draft_quantity_count(self, product_template_ids, product_variant_id
         if not self.user_has_groups('stock.group_stock_manager'):
             return res
         domain = self._product_domain(product_template_ids, product_variant_ids)
-        company = self.env['stock.location'].browse(wh_location_ids).mapped('company_id')
+        company = self.env['stock.location'].browse(wh_location_ids[0]).company_id
         svl = self.env['stock.valuation.layer'].search(domain + [('company_id', '=', company.id)])
+        quants = self.env['stock.quant'].search([('company_id', '=', company.id),
+                                                 ('product_id.product_tmpl_id', 'in', product_template_ids),
+                                                 ('location_id', 'in', wh_location_ids),
+                                                 ('product_id.product_tmpl_id', 'in', product_template_ids if product_template_ids else product_variant_ids)])
```

Comment:
this domain is wrong, no ?
```
domain = [('company_id', '=', company.id),
                                                 ('product_id.product_tmpl_id', 'in', product_template_ids),
                                                 ('location_id', 'in', wh_location_ids)]
if product_variant_ids:
     domain += [('product_id', 'in', product_variant_ids))]

```

Reply:
Yes, you're right, i made that mistake when refactoring this fix i made 2 months ago, my bad. Thank you ! I've pushed an amended commit.